app/views/incomes/BonusPaymentAmountView.scala.html

```diff
@@ -33,33 +39,57 @@
         formForErrorSummary = Some(bonusOvertimeAmountForm),
         displayBackLink = true,
         mainHeadingText = messages("tai.bonusPaymentsAmount.title",TaxYearRangeUtil.currentTaxYearRangeBetweenDelimited),
-        preHeadingText = Messages("tai.bonusPaymentsAmount.preHeading", employer.name)
+        preHeadingText = Messages("tai.bonusPaymentsAmount.preHeading", employer.name),
+        showGdsVersion = true
     )
 }
 
+
+@hint = { <p>@messages("tai.bonusPaymentsAmount.hint")</p> }
+
+
+
 @mainTemplate(
     title = messages("tai.bonusPaymentsAmount.title",TaxYearRangeUtil.currentTaxYearRangeBetweenDelimited),
-    articleClasses = None,
     authedUser = Some(user),
-    pageHeader = Some(header),
-    employerName = Some(employer.name)
+    backLinkContent = Some(Messages("tai.back-link.upper")),
+    backLinkId = "backLink"
```

Comment:
why backLink?

Reply:
why not? tests are probably expecting id of "backLink"  :)

Reply:
should we change the defaulted one to be backLink then?

Reply:
i was thinking the same - there appears more "backlink" id then the current default one  - maybe another ticket for the backlog